pkgs/top-level/all-packages.nix

```diff
@@ -23394,6 +23394,7 @@ with pkgs;
 
   openstackclient = with python311Packages; toPythonApplication python-openstackclient;
   barbicanclient = with python311Packages; toPythonApplication python-barbicanclient;
+  designateclient = with python311Packages; toPythonApplication python-designateclient;
```

Comment:
A console_scripts entrypoint is missing in setup.cfg for designateclient
See https://github.com/openstack/python-designateclient/blob/master/setup.cfg#L30

So no executable is present in bin/designate.
That's why I didn't had designateclient in all-packages here.

Reply:
openstack just loves to be inconsistent with itself....

Reply:
but yeah, having that entry without a script to run is rather pointless 😂